### DISCUSSION TITLE
How to troubleshoot SMTP settings?

### DISCUSSION THREAD

**Discussion by axelsegebrecht** [nextcloud/all-in-one#2780] [Questions] [answered]
I am running NC AIO (latest stable) on a dedicated box on my home network. The only thing that is not working is the ability to send emails using my third-party SMTP service (SMTP2GO). 

I have tried troubleshooting steps as per the main [docs ](https://docs.nextcloud.com/server/13/admin_manual/configuration_server/email_configuration.html) and can telnet to the mail server fine from the host O/S but when I send from NC it just throws an error without leaving any trace in the logs:

> A problem occurred while sending the email. Please revise your settings. (Error: Email could not be sent. Check your mail server log)

![Basic-settings-Administration-settings-Aerouant-Cloud](https://github.com/nextcloud/all-in-one/assets/8058105/36822989-0d13-4686-9e47-d82dd7ae17f9)

- Server address: mail.smtp2go.com
- Server port: 2525 (tried the various ones they have to no avail too for SSL and TLS)
- Encryption: SSL/TLS (tried without also with negative results)
- Authentication: Yes

What else can I do to troubleshoot this?

**Comment by szaimen** [comment, marked as answer]
Hi, please ask your question at https://help.nextcloud.com.

**Comment by axelsegebrecht** [comment]
Will do. I wondered if there is anything special about those Docker containers specifically that needed to be done or if it is usually just working?

**Reply by szaimen** [reply]
No, it bundles the nextcloud/server code. In regard to the mail server feature, there is nothing different compared to a "normal" installation.

**Reply by axelsegebrecht** [reply]
Thanks. Is there a way to edit the config.php in a persistent way so I can enable debugging further?

**Reply by szaimen** [reply]
https://github.com/nextcloud/all-in-one#how-to-edit-nextclouds-configphp-file-with-a-texteditor

**Comment by chrislawso** [comment]
Hi, did you find how to resolve your issue? I ask because this week I may need to connect a nc aio system with smtp2go. Thanks

**Reply by szaimen** [reply]
Hi, please ask your question at https://help.nextcloud.com.
